src/main/java/htsjdk/beta/plugin/IOUtils.java

```diff
@@ -0,0 +1,85 @@
+package htsjdk.beta.plugin;
+
+import htsjdk.io.HtsPath;
+import htsjdk.io.IOPath;
+import htsjdk.samtools.util.RuntimeIOException;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Collectors;
+
+public class IOUtils {
```

Comment:
Yeah it could be confusing. But the alternative is to reach into the samtools package where the current one lives. Maybe we can discuss more at the next meeting since there are other cases like this.